Core.ApplicationServices/SystemUsage/ItSystemUsageService.cs

```diff
@@ -346,6 +348,7 @@ public ItSystemUsage GetById(int usageId)
                                         _relationRepository.DeleteWithReferencePreload(removedRelation);
                                         _relationRepository.Save();
                                         _usageRepository.Save();
+                                        _domainEvents.Raise(new EntityDeletedEvent<SystemRelation>(removedRelation));
```

Comment:
nej nej... det er ikke relation.. det er itsystemusage og det er en EntityModified. 

Husk hvad der er root. Vi skal ikke have events på alle de små her og det er heller ikke væsentligt. Det bliver alt for komplekst. Husk at udgangspunktet for en rebuild er ændringer på en root.

I princippet er en relation en EntityModified på hhv. To og From og når toSystem f.eks. skiftes ud så er det både på fromSystem, oldSystem og newSystem :-)